.pylintrc

```diff
@@ -91,7 +91,6 @@ enable=import-error,
 # 2. See if there's spurious ones; if so disable.
 # 3. Record above.
 # 4. Remove from this list.
-       # deprecated-method,
```

Comment:
Sorry, an extra change because I was playing with other ones, I'll reinstate it.